lib/fb/page.rb

```diff
@@ -30,12 +35,9 @@ def initialize(options = {})
     #   @name="page_fans", @description="Weekly: The
     #   number of new people who have liked your Page (Unique Users)",
     #   @value=123>,..}
-    def insights
+    def insights(options = {})
       @insights ||= begin
```

Comment:
Is memoization a good idea here? the options param can be a variety of different metrics, since dates, and periods (day, month, ..)